core/consensus/babe/src/lib.rs

```diff
@@ -629,6 +635,34 @@ impl<B: Block, C> Verifier<B> for BabeVerifier<C> where
 					auxiliary: Vec::new(),
 					fork_choice: ForkChoiceStrategy::LongestChain,
 				};
+				const NANOS_PER_SEC: u32 = 1_000_000_000;
+				let mut timestamps = self.timestamps.lock();
+				// Remainder of this block is a critical section.
+				let num_timestamps = timestamps.1.len();
+				let { median_required_blocks, .. } = self.config.0;
+				if num_timestamps as u64 >= median_required_blocks && median_required_blocks > 0 {
```

Comment:
Why not the `.max(1)` solution you had before?

Reply:
I decided to have `0` disable the feature.  This fits with the common practice of using 0 to mean “no limit” or “periodic event disabled”.

Reply:
That said, Rust has `None` for this case.